playground/mocks/data/idp/idx/user-account-unlock-success.json

```diff
@@ -0,0 +1,52 @@
+{
+  "stateHandle": "02SrPNlTAOhaU8-l7a2T1Q3wNnqJl3KByb7vLpVxgf",
+  "version": "1.0.0",
+  "expiresAt": "2021-01-13T19:31:17.000Z",
+  "intent": "LOGIN",
+  "messages": {
+      "type": "array",
+      "value": [
+          {
+              "message": "You can log in using your existing username and password",
+              "i18n": {
+                  "key": "oie.selfservice.unlock_user.success.message",
+                  "params": []
+              },
+              "class": "INFO"
+          }
+      ]
+  },
```

Comment:
`nit:` this doesn't make sense for this screen, no? Perhaps we could remove this from here? Left the same question here - https://github.com/okta/okta-signin-widget/pull/1675#issuecomment-781011942

Reply:
good catch.. I entered wrong string for the key. Updated.

Reply:
By the way the message here is correct, that we display for success.